package-lock.json

```diff
@@ -10,6 +10,9 @@
                 "demo",
                 "docs"
             ],
+            "dependencies": {
```

Comment:
I'm not sure this should be here. Please revert the changes in this file, and then re-install dependencies (`npm install`). That should regenerate this file.